sup3r/pipeline/forward_pass.py

```diff
@@ -867,19 +867,27 @@ def forward_pass_chunk(data_chunk, crop_slices,
             model_class = getattr(sup3r.models, model_class)
             model = model_class.load(*model_args, verbose=False)
 
-        data_chunk = np.expand_dims(data_chunk, axis=0)
+        if isinstance(model, sup3r.models.SpatialThenTemporalGan):
+            i_lr_t = 0
+            i_lr_s = 1
+            data_chunk = np.transpose(data_chunk, axes=(2, 0, 1, 3))
+        else:
+            i_lr_t = 3
+            i_lr_s = 1
```

Comment:
Can you explain what's happening here? No change needed just trying to trace the logic.

Reply:
Yeah so if you have a spatial model first, you want the shape to be (n_obs, spatial, spatial, n_features) where n_obs is actually n_temporal. So this piece of logic sets the index for the coarse temporal / spatial shape so that you can do a good check to make sure the output is enhanced properly.